tests/integration/tier1/checks-after-conversion/test_enabled_repositories.py

```diff
@@ -1,7 +1,6 @@
+import os
```

Comment:
```suggestion
import os
import os.path
```

Reply:
It's actually an implementation detail that allows `os.path` to work if you only do `import os` so I like to be explicit here.

Reply:
Problem is that pylint complains about `os.path` not working if we do this

Reply:
Oh I see you import both `os` and `os.path`. What is the difference here? Just to be verbose and say that we are only using path from os?

Reply:
I should remove the `import os`, I thought that something like `os.access()`  must have been used in the file and so pylint would complain about that but I see that only `os.path.exists()` is specified so `import os.path` should be enough.

The pylint error is strange... It only happens for me when run under pre-commit.  If I take what pre-commit should be running for pylint and run that `pylint -sn --rcfile=.pylintrc tests/integration/tier1/checks-after-conversion/test_enabled_repositories.py`  there is no error.

Reply:
Okay.  It looks like it's a bug in pylint-2.14.x/astroid-2.11.x.  If I change the pre-commit config to require `pylint==2.15.0, astroid==2.12.13` (the latest major release series for pylint and the earliest astroid version that it requires) then the pylint error (`[E0611 no-name-in-module] No name 'path' in module 'os'` ) goes away.

@r0x0d is there a reason to pin pylint to version 2.14.4?   Should I pin to 2.15.10 instead (the current latest)?

Reply:
`2.14.4` I think that version broke the CI in the past, IIRC. But since we are moving away from it to use pre-commit, I would say it's fine to pin to that version.